src/languages/en-US/commands/notifications.json

```diff
@@ -0,0 +1,66 @@
+{
+	"youTubeSubscriptionDescription": "Manage the server's YouTube subscriptions.",
+	"youTubeSubscriptionExtended": {
+		"usages": [
+			"set upload/live channel Channel",
+			"set upload/live message Content",
+			"add/subscribe ChannelUrl",
+			"remove/unsubscribe ChannelUrl",
+			"clear",
+			"list",
+			""
+		],
+		"extendedHelp": "Manage the YouTube subscriptions for this server.",
+		"explainedUsage": [
+			[
+				"set upload/live channel Channel",
+				"The channel to send the upload or live notifications."
+			],
+			[
+				"set upload/live message Content",
+				"The message to send in the specified channel."
+			],
+			[
+				"add/subscribe ChannelUrl",
+				"Adds a new subscription."
+			],
+			[
+				"remove/unsubscribe ChannelUrl",
+				"Removes an existing subscription."
+			],
+			[
+				"clear",
+				"Clears all existing subscriptions."
+			],
+			[
+				"list",
+				"Lists all the subscriptions."
+			]
+		],
+		"examples": [
+			"set upload channel #notifications",
+			"set upload message {{channel.name}} has uploaded a new video `{{title}}` at {{time}}, go check it out! {{url}}",
+			"add https://www.youtube.com/c/youtube",
+			"remove https://www.youtube.com/c/youtube",
+			"clear",
+			"list",
+			""
+		]
+	},
+	"youTubeSubscriptionAdd": "Subscribed to <{{channelUrl}}>.",
+	"youTubeSubscriptionRemove": "Unsubscribed from <{{channelUrl}}>.",
```

Comment:
Wouldn't it be better to use `hideLinkEmbed` in the TS code instead of wrapping the string in `<>` in JSON? Especially for other translators who may not be markdown savvy.